.github/workflows/ci.yml

```
@@ -67,3 +67,20 @@ jobs:
         run: npm ci
       - name: Run tests
         run: npm run test
+  generate-docs:
+    name: Generate docs
```

```suggestion
    name: Check docs
```
In the actions list, "generate" sounds like it might be affecting the PR, while "check" makes the role clear.